__init__.py

```diff
@@ -0,0 +1,4 @@
+import os
```

Comment:
I'd use pathlib. It's better to always prefer the high level api as long as that works.